tests/integrated/test_suite

```diff
@@ -69,35 +116,35 @@ for t in tests:
 
     start_time_ = time.time()
     # Run test, piping stdout so it is not sent to console
-    status,out = shell("./runtest", pipe=True)
+    status, out = shell("./runtest", pipe=True)
```

Comment:
Should this be `command` rather than `./runtest` as sometimes we just want to make?